.github/workflows/bulid-backend.yml

```diff
@@ -1,6 +1,10 @@
 name: Run backend build
 
-on: push
+on:
+  push:
+    branches:
+      - /refs/heads/*
+      - !main
```

Comment:
Why the "!" in front of "main"?

Reply:
eigentlich wollte ich damit verhindern, dass das build zwei mal läuft beim push in main. Aber ich habe gerade gesehen, dass wir die test beim deployment ausgeschlossen haben, insofern nehme ich das lieber wieder raus. Danke! :)